kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerConnection.kt

```diff
@@ -43,8 +48,12 @@ import org.eclipse.kuksa.util.LogTag
  */
 class DataBrokerConnection internal constructor(
     private val managedChannel: ManagedChannel,
-    private val defaultDispatcher: CoroutineDispatcher = Dispatchers.Default,
+    private val dispatcher: CoroutineDispatcher = Dispatchers.Default,
 ) {
+    val subscriptions: Set<Property>
+        get() = subscribedProperties
```

Comment:
should we return a defensive copy? 
this way it would be possible to simply cast it to a mutable set and manipulate it's content